<ui version="4.0" >
 <class>EditWidget</class>
 <widget class="QWidget" name="EditWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label1" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEdit1" />
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label2" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEdit2" />
     </item>
     <item row="1" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3" >
      <widget class="QPushButton" name="resetButton" >
       <property name="text" >
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="editwidget.qrc" >:/new/prefix1/resources/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="editwidget.qrc" >:/new/prefix1/resources/plus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="editwidget.qrc" />
 </resources>
 <connections/>
</ui>
